docs/articles/documentation/extending-testcafe/creating-custom-reporter/reporterpluginhost.md

```diff
@@ -0,0 +1,280 @@
+---
```

Comment:
reporterpluginhost is object name spelt as one word. Should I separate the words in this case?
For example, the words are not separated by a hyphen here:
https://github.com/DevExpress/testcafe/blob/master/docs/articles/documentation/using-testcafe/programming-interface/createTestCafe.md